recipes/antlr4/all/patches/conanize_runtime.patch

```diff
@@ -0,0 +1,13 @@
+--- a/runtime/CMakeLists.txt
++++ b/runtime/CMakeLists.txt
+@@ -37,8 +37,8 @@
+ add_dependencies(antlr4_static make_lib_output_dir)
+ 
+ if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+-  target_link_libraries(antlr4_shared ${UUID_LIBRARIES})
+-  target_link_libraries(antlr4_static ${UUID_LIBRARIES})
++  target_link_libraries(antlr4_shared CONAN_PKG::libuuid)
++  target_link_libraries(antlr4_static CONAN_PKG::libuuid)
```

Comment:
Didn't work:

```
100%] Linking CXX shared library /tmp/c3ipr/pr_1645_14_0_0/.conan/data/antlr4/4.8/_/_/build/fbf1a0d449585ca14e0dbf0c1b68385f5fd86390/source_subfolder/dist/libantlr4-runtime.so
/usr/bin/ld: cannot find -lCONAN_PKG::libuuid
collect2: error: ld returned 1 exit status
runtime/CMakeFiles/antlr4_shared.dir/build.make:2288: recipe for target '/tmp/c3ipr/pr_1645_14_0_0/.conan/data/antlr4/4.8/_/_/build/fbf1a0d449585ca14e0dbf0c1b68385f5fd86390/source_subfolder/dist/libantlr4-runtime.so.4.8' failed
make[3]: *** [/tmp/c3ipr/pr_1645_14_0_0/.conan/data/antlr4/4.8/_/_/build/fbf1a0d449585ca14e0dbf0c1b68385f5fd86390/source_subfolder/dist/libantlr4-runtime.so.4.8] Error 1
CMakeFiles/Makefile2:151: recipe for target 'runtime/CMakeFiles/antlr4_shared.dir/all' failed
```

Reply:
What do I need to do here?

Reply:
... figured out an alternate implementation. Now both static and shared build on Linux.